Allow to disable seconds in DurationSelector (#43689)
home-assistant/home-assistant.github.io: source/_docs/blueprint/selectors.markdown

## Duration selector

The duration selector lets you select a time duration.

> This can be helpful for, e.g., delays or offsets.

Reviewer: ```suggestion
This can be helpful, for example, for delays or offsets.
```

> days: 1 # Only when enable_day was set to true

Reviewer: ```suggestion
days: 1 # Only when enable_day is set to true
```

hours: 12
minutes: 30

> seconds: 15 # Only when enable_second is true (default)

Reviewer: ```suggestion
seconds: 15 # Only when enable_second is set to true (default)
```
